event_pubsub/event_repository.py

```diff
@@ -137,11 +138,12 @@ def insert_mpe_event(self, block_number, event_name, json_str, processed, transa
         # insert into database here
         self.connection.begin_transaction()
         try:
-            insert_query = "Insert into mpe_events_raw (block_no, event, json_str, processed, transactionHash, logIndex ,error_code,error_msg,row_updated,row_created) " \
+            insert_query = "Insert into mpe_events_raw (block_no, event, json_str, processed, transactionHash, logIndex, error_code, error_msg, row_updated, row_created) " \
```

Comment:
Can we also use this opportunity to combine all these inserts? Its only the table name thats different correct?

Reply:
Yes, it would be good to clean up the code.